content/efcore/getting-started/loading-related-data.md

```diff
@@ -0,0 +1,153 @@
+# Loading Related Data  
+ 
+> **Note** {.note} 
+> If you want to see the contents of the database or how our entities relate to each other, please refer to the [Database Reference](database-reference.md) and [Model Reference](model-reference) pages. 
+ 
+## Single Layer Inclusion 
+ 
+When loading an entity from the database, EF Core will automatically include properties that are primitive types, but non-primitive types (such as other entities) won't be included. For example, if you run the following code snippet, you will see that the Author field is null: 
```

Comment:
In the other tutorials, I think they allow errors to be thrown (cuz the wording says "Hey, this results in an error! But try it anyways!").